src/App.css

```diff
@@ -0,0 +1,59 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.calculator {
+  margin: 20px auto;
+  display: grid;
+  grid-template-rows: auto auto;
+  grid-auto-rows: 80px;
+  padding: 4%;
+  background-color: rgb(182, 182, 182);
+  width: 50%;
+  grid-row-gap: 12px;
+  border-radius: 30px;
+}
+
+.displaySection {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
- [ ] Kindly follow the design given in the project description. Following the given design is very important as you will be working for clients and will be expected to deliver exactly what they demand or desire.
- Given UI with project requirements
![image](https://user-images.githubusercontent.com/62849872/211784262-92a1f2dc-3225-48ef-8f73-7739120f83b3.png)
- Your design
![image](https://user-images.githubusercontent.com/62849872/211784403-525ffdbe-a689-45a7-bf48-ff52e9fd6da3.png)
- The gaps between the buttons needs to be fixed
- The background color where the answer is displayed needs the color fixed
- When the calculator is switched ON, the 0 should be to the RIGHT as on the design UI given.
- The button colors needs to be exact
- I don't think you need the border radius if you have to respect the given UI (For both the calculator window and the buttons)